doc/api/fs.md

```diff
@@ -70,6 +70,76 @@ the entire process until they complete--halting all connections.
 The relative path to a filename can be used. Remember, however, that this path
 will be relative to `process.cwd()`.
 
+For most `fs` module functions, the `path` or `filename` argument may be passed
+as a WHATWG [URL][] object. Only [URL][] objects using the `file:` protocol are
+supported. Use of [URL][] objects is currently still experimental.
+
+```js
+const fs  = require('fs');
+const { URL } = require('url');
+const file_url = new URL('file:///tmp/hello');
+
+fs.readFile(file_url, 'utf-8', (err,data) => {
```

Comment:
In the last commits I tried to remove everything that wasn't meaningful for the examples. So now they are quite minimalistic. That was based on your suggestion, again let me know.